microvmConfigurations/netvm/default.nix

```diff
@@ -39,16 +40,12 @@ nixpkgs.lib.nixosSystem {
       #     path = "vendorid=0x050d,productid=0x2103";
       #   }
       # ];
-      # microvm.devices = [
-      #   {
-      #     bus = "pci";
-      #     path = "0001:00:00.0";
-      #   }
-      #   {
-      #     bus = "pci";
-      #     path = "0001:01:00.0";
-      #   }
-      # ];
+      microvm.devices = [
```

Comment:
I'd like to have some toggle for this, and not enable this always by default. Also this change is NVIDIA specific but affects all other targets as well. Of course this is a bigger refactoring question that might not be addressed within the context of this PR

Reply:
That's valid toggle.